mne/gui/_coreg_gui.py

```diff
@@ -550,6 +549,22 @@ def close(self, info, is_ok):  # noqa: D102
                         "processed.", "Saving Still in Progress")
             return False
         else:
+            # store configuration, but don't prevent from closing on error
+            try:
+                set_config('MNE_COREG_GUESS_MRI_SUBJECT',
+                           str(info.object.model.guess_mri_subject),
+                           set_env=False)
+                set_config('MNE_COREG_PREPARE_BEM',
+                           str(info.object.model.prepare_bem_model),
+                           set_env=False)
+                set_config('MNE_COREG_HEAD_HIGH_RES',
+                           str(info.object.model.mri.use_high_res_head),
+                           set_env=False)
+                set_config('MNE_COREG_HEAD_OPACITY',
+                           str(info.object.mri_obj.opacity), set_env=False)
+            except Exception as error:
+                print("Error saving GUI configuration:\n%s" % (error,))
+
```

Comment:
@Eric89GXL it would be straight forward to also modify `set_config()` to accept a `dict` as first argument and write multiple entries in one call, should I?